tools/build.sh

```diff
@@ -97,16 +85,15 @@ for DIR in src/request/*/ ; do
     # create directory for request
     mkdir dist/request/$REQUEST
 
-    # get all local js files included in request's index.html, which are not in a bundle
-    LIST_JS="$(grep '<script' $DIR/index.html | grep -v 'bundle-' | grep -v -E 'web-offline.js' | cut -d\" -f2)"
+    # get all local js files included in request's index.html, which are not in a bundle, a config file
+    LIST_JS="$(grep '<script' $DIR/index.html | grep -v 'bundle-' | grep -v 'config' | grep -v -E 'web-offline.js' | cut -d\" -f2)"
 
     # concat them
     for url in $LIST_JS; do
         cat $DIR/$url >> dist/request/$REQUEST/$JS_BUNDLE
     done
 
-    replace_config_variable "CONFIG.ALLOWED_ORIGIN" "KEYGUARD_ALLOWED_ORIGIN" dist/request/$REQUEST/$JS_BUNDLE
-    replace_config_variable "CONFIG.NETWORK" "KEYGUARD_NETWORK" dist/request/$REQUEST/$JS_BUNDLE
+    replace_core_lib_url dist/request/${REQUEST}/$JS_BUNDLE
```

Comment:
The core-lib (previous CDN config) does not occur in JS files, only in HTML. This line should be removed.